ui/admin/app/styles/app.scss

```
@@ -698,3 +698,23 @@
   min-height: 1rem;
   margin: 1.125rem 0 1.5rem;
 }
+
+.mapping-list {
+  width: 100%;
+
+  &-item {
+    padding: sizing.rems(xs) 0;
+    display: grid;
+    grid-template-columns: 1fr 1fr 5%;
+    align-items: center;
+    gap: sizing.rems(l);
+
+    > * {
+      margin: 0;
+    }
+  }
+
+  > :not(:last-child) {
+    border-bottom: 1px solid var(--token-color-border-primary);
+  }
```

Simplification of the border rule / positive framing:

```suggestion
  &-item {
    border-bottom: 1px solid var(--token-color-border-primary);
    padding: sizing.rems(xs) 0;
    display: grid;
    grid-template-columns: 1fr 1fr 5%;
    align-items: center;
    gap: sizing.rems(l);

    > * {
      margin: 0;
    }
  }

  &:last-child {
    border-bottom: 0;
  }
```

---

I updated it but I personally found the previous style a bit cleaner/easier to read for me since it doesn't declare the border to set it back to 0 immediately and didn't have the border declaration in two places.

Curious what makes this style easier for you to understand